cmd/utils/flags.go

```diff
@@ -1972,6 +1972,21 @@ func EnableMinerInfo(ctx *cli.Context, minerConfig miner.Config) SetupMetricsOpt
 	}
 }
 
+func EnableNodeInfo(poolConfig core.TxPoolConfig) SetupMetricsOption {
```

Comment:
why name it `Node` Info? maybe we need more accurate one?
sunc as EnableTxPoolInfo